src/main/java/core/basesyntax/handler/impl/ReturnOperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.handler.impl;
+
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+
+public class ReturnOperationHandler implements OperationHandler {
+    @Override
+    public void apply(FruitTransaction transaction) {
+        Storage.addFruit(transaction.getFruit(), transaction.getQuantity());
```

Comment:
Directly accessing the `Storage` class and its methods from the `ReturnOperationHandler` violates the encapsulation principle. Consider adding a service layer that interacts with the `Storage` and use that service here instead.